testsuite/tests/conftest.py

```diff
@@ -73,6 +77,15 @@ def pytest_runtest_setup(item):
             for capability in mark.args:
                 if capability not in CapabilityRegistry():
                     pytest.skip(f"Skipping test because current environment doesn't have capability {capability}")
+    if item.config.getoption("--drop-sandbag"):
+        sandbag_caps = (Capability.CUSTOM_ENVIRONMENT, Capability.LOGS, Capability.JAEGER)
```

Comment:
I would remove Capability.CUSTOM_ENVIRONMENT as I believe this mark is only applied on some parameters tests and is not inherently slow or requires special deployment